requirements/core.txt

```diff
@@ -6,7 +6,7 @@
 cartopy
 #conda: proj4<5
 cf_units>=2
-cftime
+cftime!=1.0.2.1
```

Comment:
This pin was merged onto `master` by #3215 ... adding it hear for the `2.2.x` branch...